<?xml version="1.0"?>
<launch>
    <node pkg="cob_mmcontroller" type="cob_config_controller" name="arm_mmcontroller_node" ns="arm_controller" cwd="node" respawn="false" output="screen">
        <remap from="/arm_controller/cart/command" to="/cmd_vel"/>
        <param name="arm_base" value="arm_0_link" />
        <param name="arm_end_effector" value="sdh_tip_link" />
        <param name="default_control_mode" value="arm_base" />
        <param name="base_to_arm_ratio" value="0.5" />
    </node>

    <node pkg="cob_mmcontroller" type="cart_move_and_pose_record.py" name="cart_move_and_pose_record" cwd="node" output="screen"/>

</launch>
